crates/bindings-csharp/Runtime/Runtime.csproj

```diff
@@ -5,7 +5,7 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <AssemblyName>SpacetimeDB.Runtime</AssemblyName>
-    <AssemblyVersion>0.8.2</AssemblyVersion>
+    <AssemblyVersion>0.9.0</AssemblyVersion>
```

Comment:
@bfops I wonder if these should be 0.9.2 now?

@RReverser had used `0.9.0` for the `0.9.1` release but *I believe* that was only because they appeared skipped for the actual `0.9.0` release.

Reply:
As discussed in chat, we only bump one up rather than keeping those in sync with STDB itself. So it's enough to bump to 0.9.1 (or just not bump at all, as we haven't had any changes since, up to you). Definitely not 0.9.2.